src/instr.rs

```diff
@@ -128,6 +128,11 @@ define_ops!{
         const_id: ConstantId = read_u32(),
         value: &'a [u32] = read_list(),
     }
+    OpSpecConstant {
+        ty_id: TypeId = read_u32(),
+        spec_const_id: ConstantId = read_u32(),
```

Comment:
It would be great to alias a new type called `SpecConstantId` at the beginning of the file to add the correct semantic.